packages/compiler/lib/decorators.ts

```diff
@@ -81,12 +81,16 @@ export function getDoc(program: Program, target: Type): string | undefined {
 }
 
 export function inspectType(program: Program, target: Type, text: string) {
+  // eslint-disable-next-line no-console
```

Comment:
This one  (and the others in this file) should be fixed as it's a decorator in the stdlib and shouldn't write to stdio.

Reply:
What is the goal of this function then? Isn't it just to log the type?

Reply:
It is, but it needs to log it in a way that won't crash the language server.

Reply:
Could file a bug on this one, pre-existing.

Reply:
Replaced `console.log` with `console.error` and `console.dir` with `console.error + util.inspec`